main.tf

```diff
@@ -21,14 +21,24 @@ resource "aws_ses_domain_identity" "ses_domain" {
 }
 
 resource "aws_route53_record" "amazonses_verification_record" {
-  count = var.enabled && var.verify_domain ? 1 : 0
+  count = var.enabled && var.verify_route53_domain ? 1 : 0
```

Comment:
`verify_route53_domain` and `verify_cloudflare_domain` should be exclusive.
I think this should be validated with `custom-validation-rules` https://www.terraform.io/docs/configuration/variables.html#custom-validation-rules

Or is there a valid use case for using both R53 and CF? Migration?

Reply:
You are correct, Im actually about to add dnsimple also to this, following the orginial format I used, as I need it asap for work, however I will update to use custom-validation-rules over the weekend. And no there would never be a use case to use more than one provider I don't think.